src/amo/sagas/home.js

```diff
@@ -30,21 +30,17 @@ export function* fetchHomeData({
 
   const state = yield select(getState);
 
-  const recommendedSearchFilters = {
-    page_size: String(
-      isDesktopSite
-        ? LANDING_PAGE_EXTENSION_COUNT
-        : MOBILE_HOME_PAGE_EXTENSION_COUNT,
-    ),
-    promoted: RECOMMENDED,
-    sort: SEARCH_SORT_RANDOM,
-  };
-
   const recommendedExtensionsParams: SearchParams = {
     api: state.api,
     filters: {
       addonType: ADDON_TYPE_EXTENSION,
-      ...recommendedSearchFilters,
+      page_size: String(
+        isDesktopSite
+          ? LANDING_PAGE_EXTENSION_COUNT
+          : MOBILE_HOME_PAGE_EXTENSION_COUNT,
+      ),
```

Comment:
Would `page_size` always be `String(MOBILE_HOME_PAGE_EXTENSION_COUNT)` since this falls in the else statement when `isDesktop` is false?

Reply:
Good catch! I will fix that in a commit.

Reply:
It's fixed now! 😄